react/components/Onboarding/components/UI/ChannelTitleInput/index.js

```diff
@@ -0,0 +1,26 @@
+import { space } from 'styled-system';
+import { preset } from 'react/styles/functions';
+import styled from 'styled-components';
+import { mixin, inputHorizontalPadding } from 'react/components/UI/Inputs';
+import AutosizeInput from 'react-input-autosize';
+
+const ChannelTitleInput = styled(AutosizeInput).attrs({
+  f: 6,
+})`
+  max-width: 100%;
+
+  input {
+    // Note: Autosize input has box-sizing "content-box". Here we subtract
+    // padding in input and border width to account for this when calculating max-width.
+    max-width: calc(100% - ${inputHorizontalPadding} - ${inputHorizontalPadding} - 2px) !important;
+    ${mixin}
+    ${space}
+    ${preset(space, { my: 5 })}
```

Comment:
No need to re-include space with a preset. And you only need to include space if you want it to be configurable in multiple situations, otherwise pull values out of the props or just use em.

Reply:
👍